core/tests/protractor_utils/ExplorationEditorTranslationTab.js

```diff
@@ -209,6 +196,93 @@ var ExplorationEditorTranslationTab = function() {
   var audioMaterialSliderDiv = element(by.css('.md-slider-wrapper'));
   var closeAudioUploaderModalButton = element(
     by.css('.protractor-test-close-audio-upload-modal'));
+  var translationFeedback = function(index) {
+    return element(by.css('.protractor-test-feedback-' + index));
+  };
+  var translationFeedbackText = function(index) {
+    return element(by.css('.protractor-test-feedback-' + index + '-text'));
+  };
+  var translationHint = function(index) {
+    return element(by.css('.protractor-test-hint-' + index));
+  };
+  var translationHintText = function(index) {
+    return element(by.css('.protractor-test-hint-' + index + '-text'));
+  };
+  var _selectLanguage = function(language) {
+    element(
+      by.css('.protractor-test-translation-language-selector')
+    ).element(
+      by.cssContainingText('option', language)
+    ).click();
+  };
+  var stateNodeLabel = function(nodeElement) {
+    return nodeElement.element(by.css('.protractor-test-node-label'));
+  };
+
+  this.deleteAudioRecord = function() {
+    waitFor.elementToBeClickable(
+      deleteRecordButton,
+      'Delete Record button is not clickable');
+    deleteRecordButton.click();
+    waitFor.pageToFullyLoad();
+  };
+
+  this.confirmDeleteAudioRecord = function() {
+    waitFor.elementToBeClickable(
+      confirmDeleteRecordButton,
+      'Confirm Delete Record button is not clickable');
+    confirmDeleteRecordButton.click();
+    waitFor.pageToFullyLoad();
+  };
+
+  this.uploadAudioRecord = function(imgPath) {
+    waitFor.elementToBeClickable(
+      uploadAudioButton,
+      'Audio Record button is not clickable');
+    uploadAudioButton.click();
+    absPath = path.resolve(__dirname, imgPath);
+    return audioUploadInput.sendKeys(absPath);
+  };
+
+  this.saveUploadedAudio = function() {
+    waitFor.elementToBeClickable(
+      saveUploadedAudioButton,
+      'Save uploaded audio button is not clickable');
+    saveUploadedAudioButton.click();
+    waitFor.pageToFullyLoad();
+  };
+
+  this.addAudioRecord = function() {
+    waitFor.elementToBeClickable(
+      startRecordButton,
+      'Add Record button is not clickable');
+    startRecordButton.click();
+    waitFor.pageToFullyLoad();
+  };
```

Comment:
In this function you sometimes describe what's being done as adding and sometimes as starting. Can you make the terminology consistent?

Reply:
Done.